src/config.js

```diff
@@ -104,9 +108,9 @@ module.exports.setConfig = function setConfig(configData) {
     }
 
     // Use SSL
-    if (configData.disableSSL) {
+    if (configData.useSSL === false) {
```

Comment:
change to `(configData.useSSL || configData.useSSL === false)` we want to catch true as well